velox/exec/benchmarks/ExchangeBenchmark.cpp

```diff
@@ -420,42 +447,123 @@ void runBenchmarks() {
+        FLAGS_width,
+        FLAGS_num_local_tasks,
+        localPartitionStatsFlat10K,
+        localPartitionWaitStats);
     return 1;
   });
 
   folly::runBenchmarks();
-  std::cout << "flat10k: " << flat10kCounters.toString() << std::endl
-            << "flat50: " << flat50Counters.toString() << std::endl
-            << "deep10k: " << deep10kCounters.toString() << std::endl
-            << "deep50: " << deep50Counters.toString() << std::endl
-            << "struct1k: " << struct1kCounters.toString() << std::endl;
+
+  std::cout
+      << "----------------------------------Flat10K----------------------------------"
+      << std::endl;
+  std::cout << "PartitionOutput: " << partitionedOutputStatsFlat10K.toString()
+            << std::endl;
+  std::cout << "Exchange: " << exchangeStatsFlat10K.toString() << std::endl;
+
+  std::cout
+      << "----------------------------------Flat50----------------------------------"
+      << std::endl;
+  std::cout << "PartitionOutput: " << partitionedOutputStatsFlat50.toString()
+            << std::endl;
+  std::cout << "Exchange: " << exchangeStatsFlat10K.toString() << std::endl;
+
+  std::cout
+      << "---------------------------------Deep10K----------------------------------"
+      << std::endl;
+  std::cout << "PartitionOutput: " << partitionedOutputStatsDeep10K.toString()
+            << std::endl;
+  std::cout << "Exchange: " << exchangeStatsDeep10K.toString() << std::endl;
+
+  std::cout
+      << "----------------------------------Deep50----------------------------------"
+      << std::endl;
+  std::cout << "PartitionOutput: " << partitionedOutputStatsDeep50.toString()
+            << std::endl;
+  std::cout << "Exchange: " << exchangeStatsDeep50.toString() << std::endl;
+
+  std::cout
+      << "---------------------------------Struct1K---------------------------------"
+      << std::endl;
+  std::cout << "PartitionOutput: " << partitionedOutputStatsStruct1K.toString()
+            << std::endl;
+  std::cout << "Exchange: " << exchangeStatsStruct1K.toString() << std::endl;
+
+  std::cout
+      << "-------------------------------LocalFlat10K-------------------------------"
+      << std::endl;
+  std::cout << "LocalPartition: " << localPartitionStatsFlat10K.toString()
+            << std::endl;
+  printMax(
+      "Producer",
+      localPartitionWaitStats.totalProducerWaitMs,
+      localPartitionWaitStats.producerWaitMs);
+  printMax(
+      "Consumer",
+      localPartitionWaitStats.totalConsumerWaitMs,
+      localPartitionWaitStats.consumerWaitMs);
+  std::sort(
+      localPartitionWaitStats.wallMs.begin(),
+      localPartitionWaitStats.wallMs.end());
+  assert(!localPartitionWaitStats.wallMs.empty());
+  std::cout << "Wall ms: " << localPartitionWaitStats.wallMs.back() << " / "
+            << localPartitionWaitStats
+                   .wallMs[localPartitionWaitStats.wallMs.size() / 2]
+            << " / " << localPartitionWaitStats.wallMs.front() << std::endl;
```

Comment:
This one could also be more descriptive:
```
Wall ms:
    Min: <X>
    Median: <Y>
    Max: <Z>
```
I don't have a preference whether it should span multiple lines or a one-liner would be good enough, but still it's better to explicitly designate the parameters of distribution.